diff --git a/.gitmodules b/.gitmodules
index 478f69b..8350711 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,18 +1,18 @@
 [submodule "lib"]
 	path = lib
-	url = ../scopehal.git
+	url = https://github.com/glscopeclient/scopehal.git
 [submodule "doc"]
 	path = doc
-	url = ../scopehal-docs.git
+	url = https://github.com/glscopeclient/scopehal-docs.git
 [submodule "src/imgui"]
 	path = src/imgui
 	url = https://github.com/ocornut/imgui
 [submodule "src/implot"]
 	path = src/implot
-	url = ../implot.git
+	url = https://github.com/glscopeclient/implot.git
 [submodule "src/imgui-node-editor"]
 	path = src/imgui-node-editor
-	url = ../imgui-node-editor.git
+	url = https://github.com/glscopeclient/imgui-node-editor.git
 [submodule "src/ImGuiFileDialog"]
 	path = src/ImGuiFileDialog
 	url = https://github.com/aiekick/ImGuiFileDialog
