diff --git a/lib/scopehal/CMakeLists.txt b/lib/scopehal/CMakeLists.txt
index 355f71ff..d3bd25db 100644
--- a/lib/scopehal/CMakeLists.txt
+++ b/lib/scopehal/CMakeLists.txt
@@ -23,7 +23,7 @@ if(LINUX AND CMAKE_SYSTEM_VERSION MATCHES "\.fc[0-9]+\.")
 	set(FEDORA TRUE)
 endif()
 
-if(WIN32 OR FEDORA)
+if(WIN32 OR LINUX)
 	set(SPIRV_LIBRARIES SPIRV SPIRV-Tools-opt SPIRV-Tools)
 else()
 	set(SPIRV_LIBRARIES "")
@@ -34,9 +34,9 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Dst_mtim=st_mtimespec")
 endif()
 
-if(LXI_LIB AND (NOT FEDORA OR TIRPC_LIB))
+if(LXI_LIB AND (NOT LINUX OR TIRPC_LIB))
 	set(HAS_LXI true)
-	if (FEDORA)
+	if (LINUX)
 		# On Fedora, tirpc is (still?) not linked into lxi
 		set(LXI_LIBRARIES ${LXI_LIB} ${TIRPC_LIB})
 	else()
